arch/arm/boot/dts/overlays/revpi-connect-se-dt-blob-overlay.dts

```diff
@@ -0,0 +1,1792 @@
+               type = "internal";
+               number = <43>;
+            };
+            pin_define@USB_LIMIT_1A2 {
+               type = "absent";
+            };
+            pin_define@SIO_1V8_SEL {
+               type = "absent";
+            };
+            pin_define@PWML {
+               type = "internal";
+               number = <41>;
+            };
+            pin_define@PWMR {
+               type = "internal";
+               number = <40>;
+            };
+            pin_define@SAFE_MODE {
+               type = "absent";
+            };
+            pin_define@SD_CARD_DETECT {
+               type = "absent";
+            };
+            pin_define@ID_SDA {
+               type = "internal";
+               number = <0>;
+            };
+            pin_define@ID_SCL {
+               type = "internal";
+               number = <1>;
+            };
+            pin_define@DISPLAY_I2C_PORT {
+               type = "internal";
+               number = <0>;
+            };
+            pin_define@DISPLAY_SDA {
+               type = "internal";
+               number = <44>;
+            };
+            pin_define@DISPLAY_SCL {
+               type = "internal";
+               number = <45>;
+            };
+         }; // pin_defines
+      }; // pins
+
+      pins_cm3 { // Pi 3 CM3
+         pin_config {
+            pin@default {
+               polarity = "active_high";
+               termination = "pull_down";
+               startup_state = "inactive";
+               function = "input";
+            }; // pin
+            pin@p14 { function = "uart0";  termination = "no_pulling"; drive_strength_mA = < 8 >; }; // TX uart0
+            pin@p15 { function = "uart0";  termination = "pull_up"; drive_strength_mA = < 8 >; }; // RX uart0
+            pin@p46 { function = "input";   termination = "pull_up";    }; // SMPS_SCL
+            pin@p47 { function = "input";   termination = "pull_up";    }; // SMPS_SDA
+            pin@p48 { function = "sdcard"; termination = "pull_up";    drive_strength_mA = < 8 >; }; // SD CLK
+            pin@p49 { function = "sdcard"; termination = "pull_up";    drive_strength_mA = < 8 >; }; // SD CMD
+            pin@p50 { function = "sdcard"; termination = "pull_up";    drive_strength_mA = < 8 >; }; // SD D0
+            pin@p51 { function = "sdcard"; termination = "pull_up";    drive_strength_mA = < 8 >; }; // SD D1
+            pin@p52 { function = "sdcard"; termination = "pull_up";    drive_strength_mA = < 8 >; }; // SD D2
+            pin@p53 { function = "sdcard"; termination = "pull_up";    drive_strength_mA = < 8 >; }; // SD D3
+            pin@p128 { function = "input";  termination = "no_pulling"; polarity = "active_low"; }; // Hotplug
+            pin@p129 { function = "output"; termination = "no_pulling"; polarity = "active_low"; }; // EMMC_ENABLE_N
+            pin@p4 { function = "output"; termination = "no_pulling"; startup_state = "inactive"; }; // RevPi Connect GPIO Mux
```

Comment:
The pin configuration for the SPI0 CLK is still needed for the ConBridge!
```suggestion
            pin@p4 { function = "output"; termination = "no_pulling"; startup_state = "inactive"; }; // RevPi Connect GPIO Mux
            pin@p39 { function = "spi"; termination = "no_pulling"; drive_strength_mA = < 8 >; }; // RevPi Connect SPI0 CLK
```

Reply:
FYI: _The dt-blob.bin is used to configure the binary blob (VideoCore) at boot time. It is not currently used by the Linux kernel, but a kernel section will be added at a later stage, when we reconfigure the Raspberry Pi kernel to use a dt-blob for configuration. The dt-blob can configure all versions of the Raspberry Pi, including the Compute Module, to use the alternative settings_
https://www.raspberrypi.com/documentation/computers/configuration.html#sections-of-the-dt-blob